tests/testthat/helper.R

```diff
@@ -0,0 +1,3 @@
+library(jsonlite)
+
+wipe_results <- function() unlink(test_path("results/"), recursive = TRUE)
```

Comment:
Do we want to be more targeted with how we use this? We could remove specific results when we need, or everything on teardown and/or setup, if we like. I worry we're going to have to go figure out where this is getting called and comment it out if we want to see the results for debugging.

Also jsonlite is not necessary anymore

Reply:
Ok @jonkeane is doing this here: https://github.com/ursacomputing/arrowbench/pull/92